tools/warnings_count/warnings_build_current.txt

```diff
@@ -1,2 +1,6 @@
 cc: Warning: -mips3 should not be used for ucode 32-bit compiles
 cc: Warning: -mips3 should not be used for ucode 32-bit compiles
+[01m[Ksrc/code/z_fireobj.c:[m[K In function ‘[01m[KFireObj_UpdateStateTransitions[m[K’:
+[01m[Ksrc/code/z_fireobj.c:110:28:[m[K [01;35m[Kwarning: [m[Kleft-hand operand of comma expression has no effect [[01;35m[K-Wunused-value[m[K]
+  110 |         ((fire->yScale * (0[01;35m[K,[m[K 6500.0f)) < (sp44 - fire->position.y))) { // Fake but IDK what else
+      |                            [01;35m[K^[m[K
```

Comment:
Does this warning still exist? If it doesn't, then remember to update this file (just ensuring this file isn't being updated should be enough)

Reply:
I re ran the warnings script and I guess it didn't work.  is it OK to remove it manually?

Reply:
Yeah. Worst case scenario Jenkins would be mad

Reply:
What happened with this?